translate/rus.json

```diff
@@ -59,7 +59,7 @@
 "book every":"книга кажды",
 "Book FAQ":"Книги ЧЗВ",
 "bottom":"дно",
-"brackets":"скобки",
+"brackets":"древо",
```

Comment:
This is just literation thing. This is used to show Cup's playoff tree, right? The new translation just calls it a 'tree'. The 'old' (here only, it is still used in modern-day Russian) meaning was for ( and ) as a term (mainly mathematical).